cli/popper/gha.py

```diff
@@ -53,6 +55,61 @@ def __init__(self, wfile, workspace):
         for e in dict(self.env):
             self.env.update({e.replace('GITHUB_', 'POPPER_'): self.env[e]})
 
+    def validate(self):
+        """
+        Validates the .workflow file
+        """
+        if not self.wf.get('workflow', None):
+            pu.fail('Workflow block must be present\n')
+        for _, wf_block in dict(self.wf['workflow']).items():
+            if not wf_block.get('resolves', None):
+                pu.fail('[resolves] attribute must be present\n')
+            else:
+                if not isinstance(
+                        wf_block['resolves'],
+                        str) and not isinstance(
+                        wf_block['resolves'],
+                        list):
+                    pu.fail('[resolves] attribute must be string or list\n')
+            if wf_block.get('on', None):
+                if not isinstance(wf_block['on'], str):
+                    pu.fail('[on] attribute must be a string\n')
+        if not self.wf.get('action', None):
+            pu.fail('Action block must be present\n')
```

Comment:
"At least one action block must be present"

Reply:
This has been fixed